.github/workflows/html-app-publish.yml

```diff
@@ -1,4 +1,4 @@
-name: Publish Docker image
+name: Publish Docker Image - HTML App
```

Comment:
Some names were adjusted to make them easier to differentiate in the Actions view.